src/main/java/com/kodilla/ecommercee/entity/GroupProduct.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.ecommercee.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class GroupProduct {
```

Comment:
Proponuje to nazwać ProductGroup zamiast GroupProduct (co może też  po angielsku oznaczać "pogrupuj product")